src/main/java/com/iridium/iridiumskyblock/api/IridiumSkyblockAPI.java

```diff
@@ -47,7 +43,10 @@ private IridiumSkyblockAPI(@NotNull IridiumSkyblock iridiumSkyblock) {
      * @return the instance of this api
      * @since 3.0.0
      */
-    public static @NotNull IridiumSkyblockAPI getInstance() {
+    public static IridiumSkyblockAPI getInstance() {
```

Comment:
Because I can't comment on outdated code:
```suggestion
    public static @Nullable IridiumSkyblockAPI getInstance() {
```